blog-article.html

```diff
@@ -0,0 +1,454 @@
+                        </li>
+                    </ul>
+                    <ul class="pointer-list">
+                        <li>
+                            List name 1
+                        </li>
+                        <li>
+                            List name 2
+                        </li>
+                        <li>
+                            List name 3
+                        </li>
+                        <li>
+                            List name 4
+                        </li>
+                    </ul>
+                    <div class="blockquote">
+                        <blockquote>
+                            The first huge metaverse arena to arrange live sports, hang out and even make bets and wagers
+                        </blockquote>
+                        <cite>
+                            <span>8,5</span>/
+                            <span>10</span>
+                            Markout Corporation
+                        </cite>
+                    </div>
+                    <div>
+                        <figure>
+                            <img src="img/dragon2.jpg" width="448" height="390" alt="dragon-face">
+                        <figcaption>Photo caption</figcaption>
+                        </figure>
+                        <figure>
+                            <img src="img/dragon3.jpg" width="448" height="390" alt="dragon-big">
+                            <figcaption>Photo caption</figcaption>
+                        </figure>
+                    </div>
+                    <h3>NFTs are digital assets that represent real-world objects</h3>
+                    <p>
+                        Setting up your avatar and personalizing your nickname on Binance is quick and easy. Users can go to binance.com under Dashboard > Settings > My Profile > Avatar and Nickname. For those of you who own NFT assets, you can select the NFT you want to use from your collection as your avatar to personalize your profile. For users who do not own any NFT avatars, check out the NFT avatar collections page to get started.
+                    </p>
+                    <p>
+                        Alternatively, if you’re on nft.binance.com, go to User Center > Settings > Basic > Set Avatar and Nickname, and you’ll be redirected to the main site. Binance NFT Creators have the option of setting their Avatars on nft.binance.com. Once you set your new unique nickname, it will be standardized and used across the Binance ecosystem.
+                    </p>
+                    <h3>
+                        NFT Avatar Generators
+                    </h3>
+                    <p>
+                        You can find a range of NFT marketplaces where you can buy existing NFTs or list your own NFTs for sale.
+                    </p>
+                </article>
+                <aside class="aside">
```

Comment:
Theoretically, it is possible to use the aside tag here. Perhaps we can replace it with something else, since it does not look quite like that, which is responsible for aside, that is, not from the side. In any case, we will try to learn more to explain why we use this or that tag and if there are arguments in the direct of aside, let's discuss them

Reply:
I know. I'd like make this block as plain div. But I asked question to Andrey Gurulev and he advised me do this like aside.

Reply:
It's really cool that you're asking questions and that more than just a div is used here 🔥 

If it would be convenient for you, I would suggest sometimes discussing what has been done and how.

For example, I would recommend making a footer tag for artice here. because we have an article and it has certain tags that are in the footer of this article. And now in aside, these tags do not relate to our article, which is exactly wrong logically, we divide one entity into two different ones. You can check this by asking yourself the questions: do the tags relate to the blog article? Can they exist without an article and will the meaning be clear then?

<img width="425" alt="Снимок экрана 2024-11-03 в 07 19 01" src="https://github.com/user-attachments/assets/fbe7fc6e-1f11-4f5c-a738-c1ebf2f236e8">

Reply:
Anyway, this part definitely belongs to the article, let's move it there?